Task_Position_the_Cubes/index.css

```diff
@@ -1,27 +1,65 @@
-
-body {
-  background-color: lightblue;
-}
-p{
```

Comment:
The 'margin' property should have a specific value. 'center' is not a valid value for margin.